src/website/shared/listeners/automatic_linkage.py

```diff
@@ -55,25 +55,30 @@ def build_new_links(container: Container) -> None:
         return
 
     drvs = produce_linkage_candidates(container)
-    proposal = CVEDerivationClusterProposal.objects.create(cve=container.cve)
```

Comment:
please, replace the flow by:

```
if not drvs: return
```

and keep the rest as-is, do not introduce and `if` for `drvs_through`, that's useless